src/app/services/components/ServiceSummary.vue

```diff
@@ -139,9 +139,9 @@ const status = computed(() => {
 
 const tags = computed(() => {
   if (props.service.serviceType === 'external' && props.externalService !== null) {
-    return Object.entries(props.externalService.tags).map(([label, value]) => ({ label, value }))
+    return props.externalService.tags
   } else {
-    return []
+    return null
```

Comment:
Am I right in thinking that if this always returns an array, then you don't have to do all the `Array.isArray` stuff higher up in this changeset?

https://github.com/kumahq/kuma-gui/pull/700/files#diff-ad73132c252f8aacc8adf35d8c8797f251a68e98b1c414cecba95bc63851aefeR40

Reply:
No. The checks for this are intended to accommodate the tags as they can appear on our APIs. Only when we get an array (specifically `LabelList[]`) do we skip passing `props.tags` through the `getLabels` function. The latter handles records, `null`, and `undefined` which covers the shapes now possible in `ServiceSummary`. For this code, it wouldn’t matter whether to use `null` or `[]` because the result would be the same.